File: beginner/pagina02/1041/1041.cpp
#include <iostream>

using namespace std;

int main(){

    float a, b;
    cin >> a >> b;

    if(a === 0.0 && b === 0.0){cout <<"Origem\n";}
    else if (a == 0.0){cout <<"Eixo Y\n";}
    else if (b == 0.0){cout <<"Eixo X\n";}
    else if (b > 0.0 && a > 0.0){cout <<"Q1\n";}
    else if (b > 0.0 && a < 0.0){cout <<"Q2\n";}
    else if (b < 0.0 && a < 0.0){cout <<"Q3\n";}
    else if (b < 0.0 && a > 0.0){cout <<"Q4\n";}
    
    return 0;
}

File: beginner/pagina02/1041/1041.lua
local x = io.read("*n")
local y = io.read("*n")
local ans = "Origem"

if x > 0 and y > 0 then
    ans = "Q1"
elseif x > 0 and y < 0 then
    ans = "Q4"
elseif x < 0 and y > 0 then
    ans = "Q2"
elseif x < 0 and y < 0 then
    ans = "Q3"
elseif x == 0 and y ~= 0 then
    ans = "Eixo Y"
elseif x ~= 0 and y == 0 then
    ans = "Eixo X"
end

print(ans)